handbook/engineering/security/goals.md

```diff
@@ -2,38 +2,59 @@
+**Problem:** The [Cloud team](../cloud/index.md) is working toward [private code on Sourcegraph Cloud](../cloud/index.md#private-code-on-sourcegraph-cloud). Prior to allowing users to index their own private code with Sourcegraph, we need confidence that we can proactively identify and resolve security issues. We also need to ensure that we're protected against known security flaws.
 
-### Confidence in our security model
 
-**Problem:** We need to have a high degree of confidence in our overall security before we ask paying customers to trust us with their private code.
 
-**Milestones:**
+**Milestones:** 
 
 1. We need a better way to store and access [our own secrets](https://docs.google.com/document/d/1HzO7szEm-h4fqlQOnVbcJdpDmfQiM7Rb-Tz4CMEYl-Q).
-1. We connect our [test security repository](https://github.com/sourcegraph/security-test/blob/main/README.md) to Sourcegraph Cloud and only [members who can access that repository on GitHub](https://github.com/sourcegraph/security-test/settings/access) can access that repository on Sourcegraph Cloud (i.e., [Sourcegraph organization owners](https://github.com/orgs/sourcegraph/people?query=role%3Aowner) and [@sourcegraph/security](https://github.com/orgs/sourcegraph/teams/security) members). (3.22 release)
-   - **DEPENDENCY:** We can't do this until we can have [private code on Sourcegraph Cloud](../cloud/index.md#private-code-on-sourcegraph-cloud)
-1. We advertise a bounty for each unique vulnerability that allows an unauthorized person to gain access to our [test security repository](https://github.com/sourcegraph/security-test/blob/main/README.md) on Sourcegraph Cloud. (3.23 release)
-   - **DEPENDENCY:** This milestone depends on milestone 3 of [Visibility into Sourcegraph Cloud's attack surface](#visibility-into-sourcegraph-clouds-attack-surface). We should detect and fix the obvious issues ourselves before we start to invite others to attack us with large bounties.
-1. We run a time-bound capture the flag event where there are larger bounties for being able to gain access to our [test security repository](https://github.com/sourcegraph/security-test/blob/main/README.md) on Sourcegraph Cloud. (3.24 release)
-1. We host our private [infrastructure repository](http://github.com/sourcegraph/infrastructure) on Sourcegraph Cloud. (TBD, somewhat dependent on outputs of CtF)
-1. We allow paying customers to host their private code on Sourcegraph Cloud. (TBD, somewhat dependent on outputs of CtF)
+2. We have implemented continuous scanning for known security vulnerabilities, and have resolved all impactful vulnerabilities.
+   1. Infrastructure vulnerability scans
+   2. Docker image scans
+   3. We have resolved compute vulnerabilities with a high severity, or a CVSS score of at least 4.0.
+
+
+
+### Secure Sourcegraph Cloud for SMBs
```

Comment:
Are we actively working on this goal in parallel with the previous goal? If not, and I think focus is good, then I would clearly call this out in some way (for example: putting this under an "inactive" or "future" section, or clearly documenting the dependency "not working on this until previous goal is done").

Reply:
Nope. Numbered goals, and made future section.